doc/nrf/ug_thingy91.rst

```diff
@@ -0,0 +1,262 @@
+
+.. include:: gs_programming.rst
+   :start-after: build_SES_projimport_open_start
+   :end-before: build_SES_projimport_open_end
+
+
+.. figure:: images/ses_thingy_configuration.png
+   :alt: Opening the AT Client sample
+
+   Opening the AT Client sample for the nrf9160_pca20035ns build target
+
+.. note::
+
+   The *Board Directory* folder can be found in the following location: ``ncs/nrf/boards/arm``.
+
+
+4. Click **OK** to import the project into SES. You can now work with the project in the IDE.
+
+   .. include:: gs_programming.rst
+      :start-after: build_SES_projimport_note_start
+      :end-before: build_SES_projimport_note_end
+
+#. To build the sample or an application:
+
+   a. Select your project in the Project Explorer.
+   #. From the menu, select **Build -> Build Solution**.
+      This builds the project.
+
+   You can find the output of the build, which includes the merged HEX file containing both the application and the SPM, in the ``zephyr`` sub-folder in the build directory.
+
+.. prog_extdebugprobe_start
+
+6. To program the sample or application:
+
+   a. Set the Thingy:91 SWD selection switch (**SW2**) to **nRF91** or **nRF52** depending on whether you have to program the nRF9160 SiP or the nRF52840 SoC component.
+   #. Connect the Thingy:91 to the debug out port on a 10-pin external debug probe, for example, nRF9160 DK (Development Kit), using a 10-pin JTAG cable.
+
+      .. note::
+
+         If you are using nRF9160 DK as the debug probe, make sure that **VDD_IO (SW11)** is set to 1.8 V on the nRF9160 DK.
+
+   #. Connect the external debug probe to the PC using USB.
+   #. Make sure that the Thingy:91 and the external debug probe are powered on.
+
+.. prog_extdebugprobe_end
+..
+
+   e. In SES, select **Target -> Connect J-Link**.
+   #. Select **Target -> Download zephyr/merged.hex** to program the sample or application onto Thingy:91.
+   #. The device will reset and run the programmed sample or application.
+
+.. _build_pgm_cmdline:
+
+Building and programming using Command line
+===========================================
+
+Complete the command line build setup (see the instructions for :ref:`Windows<build_environment_cli_win>` and :ref:`Linux<build_environment_cli_linux>`) before you start building |NCS| projects on the command line.
+
+To build and program the source code from the command line, complete the following steps:
+
+1. Open a terminal window.
+#. Go to the specific sample or application directory.
+   For example, the folder path is ``ncs/nrf/applications/asset_tracker`` when building the source code for the :ref:`asset_tracker` application on the nRF9160 SiP component and ``ncs/nrf/samples/usb/usb_uart_bridge`` when building the source code for the :ref:`usb_uart_bridge_sample` sample on the nRF52840 SoC component.
+
+#. Make sure that you have the latest version of |NCS| repository by pulling the |NCS| repository, `fw-nrfconnect-nrf`_ on GitHub using the command:
```

Comment:
@joakimtoe This sentence only makes sense if we want people to work on master. Do we? Then we should also point this out.

Reply:
Ahh, you are correct. Its up to the developer to decide if he wants to base his work on a tag or master. I suggest we guide in both scenarios.

Reply:
Replaced the existing text with : 
Make sure that you have the required version of |NCS| repository by pulling the |NCS| repository, `fw-nrfconnect-nrf`_ on GitHub using the procedures described in :ref:`dm-wf-get-ncs` and :ref:`dm-wf-update-ncs`.